script.rb

```diff
@@ -13,39 +13,49 @@ def get_binding
 
 namespace :android do
   task :ask_stuff do
-    set :output_folder, i.ask('Relative path to output? ')
+    if ARGV.length == 3
+      set :output_folder, ARGV[1]
+      project_name = ARGV[2]
+      sanitized_name = project_name.gsub(/[^A-Za-z0-9]/, '')
+      package = "mobi.lab.#{sanitized_name.downcase}"
+      set :project_name, project_name
```

Comment:
Should add empty/blank checks here, too? I assume I can enter " ", " ", " " as arguments and it would make sense to return an error if that is the case